src/utils/three-utils.js

```diff
@@ -165,6 +165,8 @@ export function cloneObject3D(source, preserveUUIDs) {
       return;
     }
 
+    clonedNode.onBeforeRender = sourceNode.onBeforeRender;
```

Comment:
Note for reviewers: Progressive loading is fired in `onBeforeRender`. So it should be copied in cloning.